public/assets/sorted-custom-node-map.json

```diff
@@ -2599,5 +2598,6 @@
   "Perlin Power Fractal Latent (PPF Noise)": 1,
   "Cross-Hatch Power Fractal (PPF Noise)": 1,
   "Linear Cross-Hatch Power Fractal (PPF Noise)": 1,
-  "CLIPTextEncodeControlnet": 1
+  "CLIPTextEncodeControlnet": 1,
+  "Load3D": 1
```

Comment:
You don't need to update this list as it is automatically generated, and not supposed to be up-to-date.

Reply:
removed